Fix compile problem when rtw_byteorder.h and asm/byteorder.h gets 
included in addition for example indirectly, do not use realtek own copy 
of the byteorder headers.

--- a/include/drv_types.h
+++ b/include/drv_types.h
@@ -25,7 +25,7 @@
 #include <drv_conf.h>
 #include <basic_types.h>
 #include <osdep_service.h>
-#include <rtw_byteorder.h>
+#include <asm/byteorder.h>
 #include <wlan_bssdef.h>
 #include <wifi.h>
 #include <ieee80211.h>
